examples/tutorials/translations/german/Part 01 - The Basic Tools of Private Deep Learning.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<h1>Teil 0: Setup&nbsp;</h1><p> </p><p>Zu Beginn müssen Sie sicherstellen, dass Sie die richtigen Dinge installiert haben. Gehen Sie dazu zum PySyft Readme und folgen Sie den Anweisungen zum Einrichten. Der einfachste Weg für die meisten ist:&nbsp;</p><ul><li>Installieren Sie Python 3.6 oder höher&nbsp;</li><li>Installieren Sie PyTorch 1.3&nbsp;</li><li>Clone PySyft (git clone&nbsp;<a href="https://github.com/OpenMined/PySyft.git" target="_blank">https://github.com/OpenMined/PySyft.git</a>)&nbsp;</li><li>cd PySyft&nbsp;</li><li>pip install -r pip-dep/requirements.txt&nbsp;</li><li>pip install -r pip-dep/requirements_udacity.txt&nbsp;</li><li>python setup.py install udacity&nbsp;</li><li>python setup.py test&nbsp;</li></ul><p>Wenn ein Teil davon für Sie nicht funktioniert (oder einer der Tests fehlschlägt), überprüfen Sie zuerst die&nbsp;<a href="https://github.com/OpenMined/PySyft.git" target="_blank">README</a>&nbsp;-Installationshilfe und öffnen Sie dann ein GitHub-Problem oder schreiben Sie den #beginner Channel in unserem Slack!&nbsp;<a href="http://slack.openmined.org/" target="_blank">slack.openmined.org</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/3178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>